src/platform/packages/private/kbn-grid-layout/grid/grid_row/grid_row.tsx

```diff
@@ -156,22 +155,25 @@ export const GridRow = ({
     );
   }, [panelIds, gridLayoutStateManager, renderPanelContents, rowIndex]);
 
+  const toggleIsCollapsed = useCallback(() => {
+    const newLayout = cloneDeep(gridLayoutStateManager.gridLayout$.value);
+    newLayout[rowIndex].isCollapsed = !newLayout[rowIndex].isCollapsed;
+    gridLayoutStateManager.gridLayout$.next(newLayout);
+  }, [rowIndex, gridLayoutStateManager.gridLayout$]);
+
   return (
     <div
-      css={css`
-        height: 100%;
-      `}
-      className="kbnGridRowContainer"
+      id={`kbnGridLayoutRow--${rowIndex}`}
+      css={styles.fullHeight}
+      className={classNames('kbnGridRowContainer', {
+        'kbnGridRowContainer--collapsed': isCollapsed,
+      })}
```

Comment:
Using a class name here is more performant so that things like the accordion's open/closed arrow can be controlled with raw CSS without a React re-render